anonlink/solving/_multiparty_solving.pyx

```diff
@@ -6,27 +7,62 @@ from anonlink.solving._multiparty_solving_inner cimport (
     Record, Group, greedy_solve_inner)
 
 
-def greedy_solve_native(candidates):
-    """Select matches from candidate pairs using the greedy algorithm.
```

Comment:
`greedy_solve_native` is moved down as a special case of `probabilistic_greedy_solve_native`.